src/app/components/displayQuestion.js

```diff
@@ -0,0 +1,35 @@
+export class DisplayQuestion {
+    constructor() {
+        this.questionNumber = 1;
+        this.questionWrapper;
+        this.questionParagraph;
+    }
+
+    generateQuestion(category) {
```

Comment:
Wydaje mi się, że lepiej byłoby przyjmować `questionNumber` jako parametr `generateQuestion( {questionNumber} )`. Natomiast `category` można przyjąć w konstruktorze i tam przypisać do  `this.questionText` tekst pytania, które ma być wyświetlane, np.
```js
constructor(category) {
  switch (category) {
      case 'people':
          this.questionText = 'Who is this character?';
          break;
      case 'vehicles':
          this.questionText = 'What is this vehicle?';
          break;
      case 'starships':
          this.questionText = 'What is this starship?';
          break;
      default:
          throw new Error(`Choose one of people, vehicles, starships`)
          break;
  }
...
}
```
A później przy tworzeniu:
```js
generateQuestion( {questionNumber} ) {
  ...
  this.questionParagraph.textContent = `${questionNumber}. ${this.questionText}`
  ...
}
```
Taki interfejs klasy bardziej pasuje do naszych wymagań, ponieważ po wyborze trybu gry ustalona jest kategoria, a zmieniają się tylko numery pytań.

Jeżeli chciałbyś zostawić inkrementację `questionNumber` to wypadałoby zaimplementować ten komponent w formie [Iteratora](https://en.wikipedia.org/wiki/Iterator_pattern#JavaScript), może być też taki interfejs:
```js
nextQuestion(): HTMLElement
hasNextQuestion(): boolean
```
Wtedy w konstruktorze `DisplayQuestion` przyjmowany byłby zakres (np. `startQuestionNumber:1, endQuestionNumber:10`) w którym mają być generowane pytania.